### PR TITLE
fix: fix donate link

### PR DESCRIPTION
**Pull request by amareshsm**
**Description:**

Fixed Donate link

### CODE REVIEW

File: src/_data/navigation.json [hunk 1]
```diff
@@ -20,5 +20,6 @@
       "text": "Playground",
       "url": "/play"
     }
-  ]
+  ],
+  "donate": "/donate"
```
Comment (nzakas) [inline src/_data/navigation.json:24]: If we are going to add this here, we should probably have both the URL and the text like we do for the other links.

Comment (amareshsm) [reply src/_data/navigation.json:24]: Can we change it like this?
```
{
"items" : [...]
"donate" : { text: "donate", URL: "/donate" }
}

```
HTML: 
```
<a href="{{ navigation.donate.url }}" class="c-btn c-btn--primary donate-link">{{navigation.donate.text}}</a>
```

Comment (nzakas) [reply src/_data/navigation.json:24]: Yup, that will work.

**Review by nzakas — APPROVED** [review]
LGTM. Thanks!

File: src/_data/navigation.json [hunk 2]
```diff
@@ -20,5 +20,9 @@
       "text": "Playground",
       "url": "/play"
     }
-  ]
-}
+  ],
+  "donate": {
+    "text": "donate",
```
Comment (nzakas) [inline src/_data/navigation.json:25]:
```suggestion
    "text": "Donate",
```

**Pull request merged**